core/templates/pages/about-page/about-page.component.ts

```diff
@@ -53,10 +53,88 @@ export class AboutPageComponent {
     },
   ];
 
+  oppiaWebsiteRawBarChartData = [
```

Comment:
The data is manually entered here, should we get this from some other location? Perhaps a database?

Reply:
Actually doing this is incorrect. Let's move these values to a separate .ts file and fetch them from there. Also write validation to ensure that the values are correct (total equals to 100 and so on.)